### DISCUSSION TITLE
Getting INFO: file_cache is only supported with oauth2client<4.0.0

### DISCUSSION THREAD

**Discussion by GijsTimmers** [kuzmoyev/google-calendar-simple-api#173] [Q&A]
Hello,

I'm getting this in my log:

    INFO: file_cache is only supported with oauth2client<4.0.0

What can I do to prevent warning from popping up?

**Comment by kuzmoyev** [comment]
`oauth2client` is not used in `gcsa`. How and why do you use it? It is deprecated.

**Reply by GijsTimmers** [reply]
I use gcsa to download Google Calendar events; log snippet:

```
DEBUG: Making request: POST https://oauth2.googleapis.com/token
DEBUG: Starting new HTTPS connection (1): oauth2.googleapis.com:443
DEBUG: https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
INFO: file_cache is only supported with oauth2client<4.0.0
INFO: file_cache is only supported with oauth2client<4.0.0
DEBUG: URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/***
```

**Reply by kuzmoyev** [reply]
I got you. That's a bit misleading log message from `google-api-python-client`. It's discussed in [this issue](https://github.com/googleapis/google-api-python-client/issues/299). I'll create an issue here to fix it. Unfortunately, there's no good workaround for you with gcsa. You have two options.

1. Set a higher log level:
```
import logging

logging.getLogger('googleapicliet.discovery_cache').setLevel(logging.ERROR)
```

or

2. Get credentials manually:
```
credentials = GoogleCalendar._get_credentials(
        token_path,
        credentials_dir,
        credentials_file,
        scopes,
        save_token,
        authentication_flow_host,
        authentication_flow_port
)
gc = GoogleCalendar(credentials=credentials)
```

**Reply by GijsTimmers** [reply]
That's kind of strange, I tried to change the log level with the code you provided but it didn't change anything
